docs(helm): Added kubeVersion to documentation.

eicnix opened the pull request:
docs(helm): Added kubeVersion to documentation and examples.

jascott1 commented on docs/charts.md:
````diff
@@ -45,6 +45,7 @@ The `Chart.yaml` file is required for a chart. It contains the following fields:
 ```yaml
 name: The name of the chart (required)
 version: A SemVer 2 version (required)
+kubeVersion: A Kubernetes version the cluster needs to match (optional)
````
Can we also describe that it is a semver here? Something like:

"Specifies the compatible version(s) of Kubernetes as a SemVer version (optional)"

bacongobbler replied:
I’d change that to a “semver range”, as it’s specifying a range of known kubernetes versions this chart is compatible with.

eicnix replied:
I adjusted the wording to your comments

jascott1 commented on docs/charts.md:
````diff
@@ -45,6 +45,7 @@ The `Chart.yaml` file is required for a chart. It contains the following fields:
 ```yaml
 name: The name of the chart (required)
 version: A SemVer 2 version (required)
+kubeVersion: A SemVer range of compatible Kubernetes SemVer versions (optional)
````
Sorry for being pedantic, could we remove the second SemVer as it seem redundant.

eicnix replied:
No worries. I just wanted to be extra specific

jascott1 reviewed (changes requested):
One nit and a rebase and it should be good to go. Thanks!

jascott1 commented:
Thank you @eicnix Would you be able to squash this into one commit?

eicnix committed:
docs(helm): Added kubeVersion to documentation.
docs(helm): Added kubeVersion to documentation and examples.
Adjusted wording for the Chart.kubeVersion description
Removing redundant wording for Chart.kubeVersion description

eicnix commented:
@jascott1 I squashed the commits
